test/forms/inputs/date-input.test.js

```diff
@@ -28,61 +27,56 @@ const WrappedDateInput = (props) => {
 
 test('DateInput renders the error message when provided', () => {
   const props = { input, meta: { invalid: true, touched: true, error } }
-  const { container } = render(<WrappedDateInput {...props} />)
-  expect(within(container).getByText('input error')).toBeTruthy()
+  render(<WrappedDateInput {...props} />)
+
+  expect(screen.getByText('input error')).toBeInTheDocument()
 })
 
 test('DateInput updates the value on change', async () => {
-  render(<WrappedDateInput />)
-
-  const input = screen.getByRole('textbox', { name: 'Field' })
-
   const user = userEvent.setup()
 
-  await user.click(input)
+  render(<WrappedDateInput />)
 
-  await user.keyboard('02/02/2023{Enter}')
+  const input = screen.getByRole('textbox', { name: 'Field' })
+  await user.type(input, '02/02/2023{Enter}')
 
   expect(input).toHaveValue('02/02/2023')
 })
 
 test('DateInput sets the placeholder text correctly', () => {
   const props = { placeholderText: 'Test Placeholder' }
   render(<WrappedDateInput {...props} />)
-  const input = screen.getByRole('textbox', { name: 'Field' })
 
-  expect(within(input).findByText('Test Placeholder')).toBeTruthy()
+  expect(screen.getByPlaceholderText('Test Placeholder')).toBeInTheDocument()
 })
 
 test('DateInput invokes onChange with a Date object', async () => {
+  const user = userEvent.setup()
   const onChange = jest.fn()
   const props = { input: { ...input, onChange, onBlur: noop }, meta: {} }
+
   render(<WrappedDateInput {...props} />)
 
-  const input = screen.getByRole('textbox')
+  await user.click(screen.getByRole('textbox'))
 
-  const user = userEvent.setup()
+  const option = screen
+    .getAllByRole('option', { selected: false, hidden: false })
+    .at(0)
+  await user.click(option)
 
-  await user.click(input)
-  await user.keyboard('02/02/2023')
-  expect(input).toHaveValue('02/02/2023')
-
-  // expect(onChange).toHaveBeenCalledTimes(8)
-  expect(onChange.mock.calls[0][0] instanceof Date).toBe(true)
+  expect(onChange).toHaveBeenCalledTimes(1)
+  expect(onChange).toHaveBeenCalledWith(expect.any(Date))
 })
 
 test("DateInput defaults tabbable item to today's date", async () => {
-  const props = { input: { ...input, value: '' }, meta: {} }
-  render(<DateInput {...props} />)
-  const input = screen.getByRole('textbox')
-
   const user = userEvent.setup()
+  const props = { input: { ...input, value: '' }, meta: {} }
 
-  await user.click(input)
+  render(<DateInput {...props} />)
 
+  const dateInput = screen.getByRole('textbox')
+  await user.click(dateInput)
   const current = screen.getByRole('option', { current: 'date' })
-  const today = moment().format('dddd, MMMM Do, YYYY')
-  const currentDateAriaALabel = screen.getByLabelText(`Choose ${today}`)
-  expect(current).toBeTruthy()
-  expect(currentDateAriaALabel).toBeTruthy()
+
+  expect(current).toHaveProperty('tabIndex', 0)
```

Comment:
I don't think this assertion adequately covers the scope test case. It confirms that the `current` item has a tab index of 0, but it **does not** confirm that the value is equal to today's date.